pkg/internalapis/cnsvolumeinfo/cnsvolumeinfoservice.go

```diff
@@ -97,6 +104,29 @@ func InitVolumeInfoService(ctx context.Context) (VolumeInfoService, error) {
 	return volumeInfoServiceInstance, nil
 }
 
+// ListAllVolumeInfoes lists all the VolumeInfo CRs present in the cluster
+func (volumeInfo *volumeInfo) ListAllVolumeInfoes() []interface{} {
+	volumeInfoCrs := volumeInfo.volumeInfoInformer.GetStore().List()
+	return volumeInfoCrs
+}
+
+// VolumeInfoCrExistsForVolume returns true if VolumeInfo CR for
+// a given volume exists
+func (volumeInfo *volumeInfo) VolumeInfoCrExistsForVolume(ctx context.Context, volumeID string) (bool, error) {
+	log := logger.GetLogger(ctx)
+
+	key := csiNamespace + "/" + volumeID
+	_, found, err := volumeInfo.volumeInfoInformer.GetStore().GetByKey(key)
+	if err != nil {
+		return false, logger.LogNewErrorf(log, "failed to find vCenter for VolumeID: %q", volumeID)
+	}
+	if !found {
+		log.Debugf("VolumeInfo CR for volume %s not found", volumeID)
+		return false, nil
+	}
```

Comment:
You can merge this.

Reply:
Could you elaborate how? Both if conditions are returning different parameters.